src/Fields/Media.php

```diff
@@ -139,6 +146,52 @@ private function setOrder($ids)
         $mediaClass::setNewOrder($ids);
     }
 
+    private function addExistingMedia(NovaRequest $request, $data, HasMedia $model, string $collection, Collection $medias): Collection
+    {
+        $added_media_ids = $medias->pluck('id')->toArray();
```

Comment:
I'd strongly suggest you to use PSR standards as far as possible. In this case: using camel-case for variable names in php. (all php files in this PR)

Reply:
PSR coding standards don't specify variable naming convention (correct me if I'm wrong), but I prefer to be consistent so I'll update it :)